sdk/mediaservices/microsoft-azure-media/pom.xml

```diff
@@ -171,16 +171,6 @@
         </executions>
       </plugin>
 
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.10.1</version> <!-- {x-version-update;org.apache.maven.plugins:maven-compiler-plugin;external_dependency} -->
-        <configuration>
-          <source>1.7</source>
-          <target>1.7</target>
-        </configuration>
```

Comment:
Same here - what version will the source and target be if this is removed?

Reply:
Changing the baseline will break customers if they are still on Java 7. We should not change the baseline for track 1. We either deprecate when an equivalent track 2 lib is available or continue to support existing track 1 libraries.

Reply:
Reverted this change.